libs/openFrameworks/video/ofVideoPlayer.cpp

```diff
@@ -403,7 +403,9 @@ void ofVideoPlayer::resetAnchor(){
 
 //------------------------------------
 void ofVideoPlayer::draw(float _x, float _y, float _w, float _h) const{
-	ofGetCurrentRenderer()->draw(*this,_x,_y,_w,_h);
+	if( isUsingTexture() && getTexture().isAllocated() ){
+		ofGetCurrentRenderer()->draw(*this,_x,_y,_w,_h);
+	}
```

Comment:
this check belongs in the renderer, if we use a renderer that doesn't use the texture it won't work like this. you could draw a video with cairo, doesn't make much sense :) but you can do it, and this would break it without producing any error either